package-lock.json

```diff
@@ -0,0 +1,2723 @@
+{
```

Comment:
This project uses yarn, not npm. We already have a yarn.lock file and should not add a package-lock.json file. Please remove this.